glusterd2/commands/volumes/volume-smartvol-txn.go

```diff
@@ -61,20 +56,19 @@ func txnPrepareBricks(c transaction.TxnCtx) error {
 				return err
 			}
 
-			dev := "/dev/" + b.VgName + "/" + b.LvName
 			// Make Filesystem
-			err = deviceutils.MakeXfs(dev)
+			err = deviceutils.MakeXfs(b.DevicePath)
 			if err != nil {
-				c.Logger().WithError(err).WithField("dev", dev).Error("mkfs.xfs failed")
+				c.Logger().WithError(err).WithField("dev", b.DevicePath).Error("mkfs.xfs failed")
 				return err
 			}
 
 			// Mount the Created FS
-			err = deviceutils.BrickMount(dev, brickMountDir)
+			err = deviceutils.BrickMount(b.DevicePath, b.Mountdir)
 			if err != nil {
 				c.Logger().WithError(err).
-					WithField("dev", dev).
-					WithField("path", brickMountDir).
+					WithField("dev", b.DevicePath).
```

Comment:
question:

which one is best to use ```WithFields``` or ```WithFiled``` in this case

Reply:
I will change this to WithFields